src/Main.elm

```diff
@@ -0,0 +1,104 @@
+module Main exposing (..)
```

Comment:
You should not expose all of your main.
Think about what will be useful outside of this module.
For example, the main function won't be useful outside of this module.